tests/backup/backup_helper.go

```diff
@@ -3644,6 +3644,7 @@ func GetVolumeMounts(AppContextsMapping *scheduler.Context) ([]string, error) {
 func UpdateBackupLocationOwnership(name string, uid string, userNames []string, groups []string, accessType OwnershipAccessType, publicAccess OwnershipAccessType, ctx context.Context) error {
 	log.Infof("UpdateBackupLocationOwnership for users %v", userNames)
 	backupDriver := Inst().Backup
+	groups = append(groups, "px-admin-group")
```

Comment:
Why do we have to do this?

Reply:
By default "px-admin-group" will have ownership to all RBAC resources, and it should not be resetted if are setting the groups list to null or any other groups.

Reply:
Then in this case, you need to get the ownership group from the RBAC object first and then apply the same.
Let's say I have set another group (say `super-group`) along with `px-admin-group` as the owner group, now if we just add `px-admin-group`, we are effectively removing the `super-group` right?
Also, I feel the same has to be done with the users.

Reply:
There is challenge in implementing that approach in api way.
For eg: suppose the rbac object has current ownership as super-group , and now user want to cancel the ownership (nil), This method wont be useful as it will always fetch the current ownership and applies it. 
UI does this workflow by keeping a seperate list of current ownerships(users and groups), And that list is passed to px-backup for applying the ownership while updation of rbac object.

Reply:
As discussed will change this UpdateOwnership to AddOwnership and RemoveOwnership. Since the change requires changes in existing tests where it is used and all rke uses cloudCredentialUpdation. it require more time for sanity check. will take it in seperate ticket https://portworx.atlassian.net/browse/PA-1678

@mkoppal-px